x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/VerifierTests.java

```diff
@@ -86,7 +86,8 @@ public void testUnsupportedAndMultiTypedFields() {
             new EsIndex(
                 "test*",
                 Map.of(unsupported, unsupportedField, multiTyped, multiTypedField, "int", unsupportedField, "double", multiTypedField)
-            )
+            ),
+            ipIndices
```

Comment:
Should this have all six indices in it? Maybe it doesn't matter, but more realistic I think.